src/views/Home.vue

```diff
@@ -84,21 +35,18 @@
         border-radius: 10%;
         max-width: 50%;
     }
+    #goToScopeButton {
+        background-color: #3333ff;
```

Comment:
This is now a "magic number": A color setting that just _coincidentally_ happens to match the same value as used in Footer.vue. If the design intent is for them to be the same color (which I have no objection to) then the CSS must be reorganized so that they are the same color by virtue of both referring to the same color setting in our source files. We can and should avoid any instances of being obliged to make parallel changes to identical information.
TL;DR: Make this setting DRY. ;-)

Reply:
Added CSS variables for colors in https://github.com/numberscope/frontscope/pull/142/commits/34c2515d3fee076e543ff791c405a06fe109aad4.

Reply:
The usual pattern I've seen is to provide such var()s with functional designations, like "header background" or "highlight color". That guides where in the implementation to use them, and what the effects will be if they are changed. Looked at another way, you cant'really make "light blue" be fuchsia, but you could make "keywords" be fuchsia,  for example. Or another way, right now it still looks like coincidences that certain elements are both light blue, rather than the design intent captured in the semantics of the items being used. Should we adopt this pattern?

Reply:
I apologize for not understanding your suggestion initially.

I think I understand it now. You would like the CSS variables to indicate the design intent of the elements they are applied to, correct?

If the CSS variables indicate the design intent of the elements they are applied to, then it seems like less of a coincidence that they have the same color, for instance. Is that the idea?

If that's the idea, I think it's a good one.

I can't really think of what we would name them other than `light-background-color` and `dark-background-color`. The light blue is used in the navbar and the background color for the buttons on the scope when you hover over them. The dark blue is used in the footer and the "Go to the 'Scope" button.

Did I understand your suggestion correctly? Do you have suggestions for the names?

Reply:
OK, I ended up with --ns-foreground for the default text color of the entire app, and --ns-navigation-{background,foreground} for the navbar and other navigation-like elements, such as button-like things when you are over them, and --ns-information-{background,foreground} for message-y type items, like the footer and the BundleManager reports when a sequence or visualizer is chosen. I also tried to make all button-ish things look and behave the same: dark rounded border, no special colors for the content when not over them, but the nav colors when hovering over them. I wasn't entirely sure that the visualizer and sequence choices should be button-like, but they basically seemed to be behaving them that way, so I made them look that way, too.

I am not pretending that this is now good graphic design, but at least I feel like now it's (a) pretty consistent, and (b) will be pretty easy to change if we improve the visual design while keeping it consistent. Let me know what you think.